packages/cli/src/ui/components/messages/ToolConfirmationMessage.tsx

```diff
@@ -56,6 +56,8 @@ export const ToolConfirmationMessage: React.FC<
     onConfirm(outcome);
   };
 
+  const isTrustedFolder = config?.isTrustedFolder() !== false;
```

Comment:
we should probably make config be non-nullable to simplify this and avoid bugs when we accidentally forget to pass in a config.

Reply:
Ok, will check and send a separate PR for that.